src/Http/Cookie/RequestCookies.php

```diff
@@ -37,33 +37,4 @@ public static function createFromRequest(ServerRequestInterface $request, $cooki
 
         return new static($cookies);
     }
-
-    /**
-     * Checks if the collection has a cookie with the given name
-     *
-     * @param string $name Name of the cookie
-     * @return bool
-     */
-    public function has($name)
-    {
-        $key = mb_strtolower($name);
-
-        return isset($this->cookies[$key]);
-    }
-
-    /**
-     * Get a cookie from the collection by name.
-     *
-     * @param string $name Name of the cookie to get
-     * @throws \InvalidArgumentException
-     * @return Cookie
-     */
-    public function get($name)
```

Comment:
Also both this method and `has()` are implemented in this class's parent. It is worth recognizing that this class will be removed very soon as well.